README.md

```diff
@@ -40,7 +40,7 @@ It requires to have the following namespaces required:
 
 For the service.invoices ns create a table:
 
-`CREATE TABLE users (id SERIAL, email VARCHAR(50))`
+`CREATE TABLE users (id SERIAL, email VARCHAR(50), authored VARCHAR(20))`
```

Comment:
this name is a bit ambigous. Better: `author`, `authored-by`, `created-by` etc

Reply:
will change to author